_data/icons/taraxa.json

```diff
@@ -0,0 +1,6 @@
+{
+  "url": "ipfs://QmQhdktNyBeXmCaVuQpi1B4yXheSUKrJA17L4wpECKzG5D",
+  "width": 512,
+  "height": 512,
```

Comment:
size seems to be wrong

identify QmQhdktNyBeXmCaVuQpi1B4yXheSUKrJA17L4wpECKzG5D         19:30:11
QmQhdktNyBeXmCaVuQpi1B4yXheSUKrJA17L4wpECKzG5D PNG 310x310 310x310+0+0 8-bit sRGB 28094B 0.000u 0:00.000

Reply:
Hmm weird, our designer named it 512. I'm addressing it now:)